thrift/thrift0.13/src/test/java/com/linecorp/armeria/common/thrift/text/TTextProtocolTest.java

```diff
@@ -139,9 +141,14 @@ void tTextNamedEnumProtocolReadWriteTest() throws Exception {
         msg1.write(new TTextProtocol(new TIOStreamTransport(baos), true));
         final byte[] bytes = baos.toByteArray();
 
+        // ignore y since it can be serialized to either 1 (<0.19) or ALPHA (>=0.19)
         assertThatJson(CR_PATTERN.matcher(baos.toString()).replaceAll(""))
                 .when(IGNORING_ARRAY_ORDER)
+                .whenIgnoringPaths("y")
                 .isEqualTo(namedEnumSerialized);
+        assertThatJson(CR_PATTERN.matcher(baos.toString()).replaceAll(""))
+                .node("y")
+                .matches(anyOf(hasToString("1"), hasToString("ALPHA")));
```

Comment:
Suggestion) Should we match the exact value depending on `:thrift` module versions?
We could read the version from `/META-INF/com.linecorp.armeria.versions.properties` in runtime.

Reply:
Updated. While doing so, I found that the thrift artifact versions weren't fetched properly due to having a `.`, so `lastIndexOf` is used in `Versions`